### PR TITLE
threshold works for a given directory

### PR DESCRIPTION
**Pull request by amattadohb**
Preliminary "thresholding" works, the threshold function converts all .dcm images in the lexicographical order of filenames into a 3d numpy array.

### CODE REVIEW

File: test/reverse_filename.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import os
+
+
+src_path = './'
+dst_path = '../good/'
+num_files = 98
+
+for i in xrange(0, num_files):
+	if i < 10:
+		src = src_path + '00000' + str(i) + '.dcm'
```
Comment (achintyagopal) [inline test/reverse_filename.py]: src = src_path + str(i).zfill(6) + ".dcm"

Comment (amattadohb) [reply test/reverse_filename.py]: yes, that's better in every way

File: test/threshold.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+def threshold(directory):
+
+	real_output_img = []
+	index = 0
+	for filename in os.listdir(directory):
+		if filename.endswith(".dcm"):
+			RefDs = dicom.read_file(directory + filename)
+			ConstPixelDims = (int(RefDs.Rows), int(RefDs.Columns))
+			ConstPixelSpacing = (float(RefDs.PixelSpacing[0]), float(RefDs.PixelSpacing[1]), float(RefDs.SliceThickness))
```
Comment (achintyagopal) [inline test/threshold.py]: You can remove third value from this tuple

File: test/reverse_filename.py (hunk 1 above)
Comment (nextBillyonair) [reply test/reverse_filename.py]: %s%06d.dcm % (src_path, i )

**Pull request merged**